CONTRIBUTING.md

```diff
@@ -100,4 +100,23 @@ Then, re-run
 ```
 pipenv sync --dev
 ```
-to install the relevant dependencies.
\ No newline at end of file
+to install the relevant dependencies.
+
+## Read the Docs
```

Comment:
I don't like the concept of modifying existing documentation while adding Read the Docs, but I don't know where else to put this.